### PR TITLE
adding html

### PR DESCRIPTION
**Pull request by miserrano7**
missing html file, adding one in

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (CandyDeLight) [inline templates/index.html:1]: good job. seems to be working.

**Comment by Mcncea** [comment]
Goooooood job

**Review by CandyDeLight — APPROVED** [review]
seems good. i'll approve.

**Review by Mcncea — COMMENTED** [review]
Looks good

**Pull request merged**